poi-ooxml/src/main/java/org/apache/poi/openxml4j/opc/OPCPackage.java

```diff
@@ -459,13 +473,10 @@ public void close() throws IOException {
             return;
         }
 
-        if (StringUtil.isNotBlank(this.originalPackagePath)) {
-            File targetFile = new File(this.originalPackagePath);
-            if (!targetFile.exists()
-                    || !(this.originalPackagePath
-                            .equalsIgnoreCase(targetFile.getAbsolutePath()))) {
+        if (this.originalPackagePath != null) {
+            if (!Files.exists(originalPackagePath)) {
```

Comment:
no Files.isSameFile call here? I know it gets called right away in save but its still a small short circuit here